tests/python/unittest/test_image.py

```diff
@@ -132,26 +132,30 @@ def test_color_normalize(self):
 
 
     def test_imageiter(self):
-        im_list = [[np.random.randint(0, 5), x] for x in TestImage.IMAGES]
-        test_iter = mx.image.ImageIter(2, (3, 224, 224), label_width=1, imglist=im_list,
-            path_root='')
-        for _ in range(3):
+        def check_ImageIter(dtype='float32'):
```

Comment:
nit: use lower case imageiter to be consistent with test_imageiter

Reply:
Done